spec/Section 2 -- Language.md

```diff
@@ -166,9 +166,9 @@ characters to support interoperation with as many other systems as possible.
 
 ## Query Document
 
-Document : Definition+
+Document :: Definition+
```

Comment:
For more detail on this, see http://facebook.github.io/graphql/#sec-Lexical-and-Syntactical-Grammar